addons/common/functions/fnc_checkFiles.sqf

```diff
@@ -107,42 +107,36 @@ if (!isServer && {_platform in ["linux", "osx"]}) then {
     // Linux and OSX client ports do not support extensions at all
     INFO("Operating system does not support extensions");
 } else {
-    {
-        private _extension = configName _x;
-        private _isWindows = _platform == "windows" && {getNumber (_x >> "windows") == 1};
-        private _isLinux = _platform == "linux" && {getNumber (_x >> "linux") == 1};
-        private _isClient = hasInterface && {getNumber (_x >> "client") == 1};
-        private _isServer = !hasInterface && {getNumber (_x >> "server") == 1};
+    private _extension = "ace";
+    private _isWindows = _platform == "windows" && {getNumber (_x >> "windows") == 1};
```

Comment:
_x is undefined, I think this whole section needs a rewrite
I'm assuming we could do linux if we wanted but I think everything we have is still client only
so something like this should cover what we need?

```
private _platform = toLowerANSI (productVersion select 6);
if (_platform in ["linux", "osx"]) then {
    // Linux and OSX client ports do not support extensions at all
    if (hasInterface) then {
        WARNING("Operating system does not support extensions");
    } else {
        INFO("Operating system does not support extensions");
    };
} else {
    ("ace" callExtension ["version", []]) params ["_versionEx", "_returnCode"];
    _versionEx = _versionEx select [0, 8]; // git hash
    INFO_1("Extension [Version: %1]",_versionEx);
    if (hasInterface && {(_returnCode != 0) || {_version == ""}}) then {
        private _errorMsg = format ["Extension not found. [Return Code: %1]", _returnCode];
        ERROR(_errorMsg);
        ["[ACE] ERROR", _errorMsg] call FUNC(errorMessage);
    };
};
```

Reply:
```sqf
("ace" callExtension ["version", []]) params ["_versionEx", "_returnCode"];
_versionEx = _versionEx select [0, 8]; // git hash
if (hasInterface && {(_returnCode != 0) || {_version == ""}}) then {
    private _errorMsg = format ["Extension not found. [Return Code: %1]", _returnCode];
    ERROR(_errorMsg);
    ["[ACE] ERROR", _errorMsg] call FUNC(errorMessage);
};
```

-> 

```sqf
("ace" callExtension ["version", []]) params ["_versionEx", "_returnCode"];

if (_returnCode != 0 || {_versionEx == ""}) then {
    private _errorMsg = format ["Extension not found. [Return Code: %1]", _returnCode];
    ERROR(_errorMsg);
    
    if (hasInterface) then {
        ["[ACE] ERROR", _errorMsg] call FUNC(errorMessage);
    };
} else {
    _versionEx = _versionEx select [0, 8]; // git hash
    INFO_1("Extension [Version: %1]",_versionEx);
};
```

Changed so that
- error message is logged for all machines, regardless if interface is present or not.
  > I think everything we have is still client only
  Imo we should keep logging for all machines.
- version message is only logged if there is no error.
- `_version` -> `_versionEx`

Otherwise your proposed changes lgtm.